utensils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wethegit/sweet-potato-utensils",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
I think this should be `1.1.0` what do you think? It adds a new feature.

Reply:
Good call.